### PR TITLE
introduce the skip-confirm flag to adapt the automation platform

### PR DESCRIPTION
**Pull request by lonng**
Signed-off-by: Lonng <heng@lonng.org>

### CODE REVIEW

File: cmd/deploy.go [hunk 1]
```diff
@@ -70,6 +71,7 @@ func newDeploy() *cobra.Command {
 	cmd.Flags().BoolVar(&opt.usePasswd, "password", false, "Specify the password of system user")
 	cmd.Flags().StringVar(&opt.keyFile, "key", "", "Specify the key path of system user")
 	cmd.Flags().StringVar(&opt.passphrase, "passphrase", "", "Specify the passphrase of the key")
+	cmd.Flags().BoolVar(&opt.skipConfirm, "skip-confirm", false, "Skip the confirmation of topology")
```
Comment (AstroProfundis) [inline cmd/deploy.go]: How about `--noconfirm` as in `pacman`  or `-y|--assumeyes` as in `yum`?

File: cmd/destroy.go [hunk 2]
```diff
@@ -80,5 +78,8 @@ func newDestroyCmd() *cobra.Command {
 			return nil
 		},
 	}
+
+	cmd.Flags().BoolVar(&skipConfirm, "skip-confirm", false, "Skip the confirmation of destroying")
```
Comment (AstroProfundis) [inline cmd/destroy.go]: ditto

File: cmd/deploy.go (hunk 1 above)
Comment (lonng) [reply cmd/deploy.go]: Nice

File: cmd/destroy.go (hunk 2 above)
Comment (lonng) [reply cmd/destroy.go]: addressed

**Pull request merged**